src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/ArgumentCompleters/AccountSkuCompleterAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Microsoft.Azure.Commands.Management.CognitiveServices.ArgumentCompleters
+{
+    using System.Management.Automation;
+
+    public class AccountSkuCompleterAttribute : ArgumentCompleterAttribute
+    {
+        public AccountSkuCompleterAttribute()
+               : base(CreateScriptBlock())
+        {
+        }
+
+        private static ScriptBlock CreateScriptBlock()
+        {
+            string scripts = @"param($commandName, $parameterName, $wordToComplete, $commandAst, $fakeBoundParameter)
+$location = $fakeBoundParameter['Location']
+$type = $fakeBoundParameter['Type']
+$values = $(Get-AzureRmCognitiveServicesAccountSkus -Location $location -Type $type) | ForEach-Object { $_.Name }
```

Comment:
@yangyuan This looks great! My only comment is that we are now only shipping Az (rather than AzureRm), so the two commands used should be Get-AzCognitiveServicesAccountSkus and Get-AzCognitiveServicesAccountType.

As a side note, we will likely be implementing a generic SKU completer, and will likely replace this implementation with the generic when created.  However, we do not have a timeframe for this, so getting the Cognitive Services specific SKU completer out there now will be great for customers.